### PR TITLE
noImplicitAny: Reduce errors to 3800-ish

### PR DESCRIPTION
**Pull request by tskarhed**
**What this PR does / why we need it**:
#14714
**Special notes for your reviewer**:
Not sure how to handle the errors with `string` as a key, so I used ts-ignore

### CODE REVIEW

File: public/app/plugins/panel/graph/jquery.flot.events.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 import _ from 'lodash';
+//@ts-ignore
 import Drop from 'tether-drop';
 
 /** @ngInject */
-export function createAnnotationToolip(element, event, plot) {
+export function createAnnotationToolip(
+  element: any[] | JQLite,
+  event: any,
+  plot: { getOptions: () => { events: { manager: any } } }
```
Comment (torkelo) [inline public/app/plugins/panel/graph/jquery.flot.events.ts]: this seems to be reused in two locations, would be nice with an interface type for it

File: public/app/plugins/panel/graph/module.ts [hunk 2]
```diff
@@ -269,24 +272,24 @@ class GraphCtrl extends MetricsPanelCtrl {
...
 
-  onToggleSort = (sortBy, sortDesc) => {
+  onToggleSort = (sortBy: any, sortDesc: any) => {
     this.panel.legend.sort = sortBy;
     this.panel.legend.sortDesc = sortDesc;
     this.render();
   };
 
-  onToggleAxis = info => {
+  onToggleAxis = (info: { alias: _.PartialDeep<unknown>; yaxis: any }) => {
```
Comment (torkelo) [inline public/app/plugins/panel/graph/module.ts]: PartialDeep unknown?

**Review by torkelo — CHANGES_REQUESTED** [review]
Nice progress! just some minor comments

File: public/app/plugins/panel/graph/module.ts (hunk 2 above)
Comment (dehrax) [reply public/app/plugins/panel/graph/module.ts]: Yeah, I have no idea. VSCode automatically did that one. I'm just changing it to `any`, as I really can't see why it should be more complex than that.

**Review by torkelo — APPROVED** [review]

**Pull request merged**